packages/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -361,6 +361,23 @@ class Transforms {
           }
         }
 
+        // Decrypt encrypted text map if present
+        if (activity.encryptedTextKeyValues !== undefined) {
+          for (let [value] of activity.encryptedTextKeyValues.values()) {
```

Comment:
My JavaScript is starting to rust. But `[value]` stands out to me. Are you sure it's what you mean to do? e.g.

<img width="372" alt="image" src="https://user-images.githubusercontent.com/5473486/203595136-7ae0e38c-1f94-424e-bfdf-af175a6901c4.png">

Reply:
Or if I use objects

<img width="362" alt="image" src="https://user-images.githubusercontent.com/5473486/203598524-5b20ccbe-5567-4b19-8893-7dbf657b5390.png">